packages/shared/src/lib/element.ts

```diff
@@ -60,6 +60,6 @@ export const parentClassContains = (
 };
 
 export const isCompanionActivated = (): boolean =>
-  !!document
```

Comment:
This caused a lot of errors in dev environment

Reply:
I rather not move this since as it has nothing to do with the PR in general 🙏